guides/kubernetes/kubernetes.md

```diff
@@ -0,0 +1,75 @@
+# Kubernetes
+
+Kubernates(also written as K8s) is an open source system for automatic , scaling and managemnt of [containerized applications](https://searchitoperations.techtarget.com/definition/application-containerization-app-containerization). It is the most popular tool among all currently present which is used mostly by all the major giants of the industry including Google who actually founded it, Amazon and Microsoft etc.
```

Comment:
Hey! Below are few suggestions, if possible try to incorporate:

Change - "kubernates" to "Kubernetes" : a typo.
Remove comma after automatic.
Change - "managemnt" to "management" : a typo.
Remove, "Amazon and Microsoft etc" from the last sentence.